src/inference/src/cpp/core.cpp

```diff
@@ -28,7 +28,15 @@ std::string find_plugins_xml(const std::string& xml_file) {
             return xml_file_name;
         }
     }
-    const auto ov_library_path = ov::util::get_ov_lib_path();
+
+    // get_ov_lib_path() returns UTF-8 encoded string when UNICODE support is enabled
+    const auto ov_library_path_str = ov::util::get_ov_lib_path();
+
+#ifdef OPENVINO_ENABLE_UNICODE_PATH_SUPPORT
```

Comment:
Could you provide some unit test for `get_ov_lib_path` with sample path which create an issue.

Errors will probably exist on windows with some special characters only.